frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/auto_completion_node_widget.dart

```diff
@@ -279,15 +345,20 @@ class _AutoCompletionInputState extends State<_AutoCompletionInput> {
           }
         },
         onEnd: () async {
+          writeEndPath =
+              widget.editorState.document.root.children.last.path.last;
```

Comment:
I suppose you can replace it with `node.path.previous`.

Reply:
In this case the correct path can be obtained only using widget.editorState.document.root.children.last.path

For example:

There is a text already written, let's say "Hello world" and the user start using AI after that text.

in that case widget.node.previous.path shows the path [1], while widget.editorState.document.root.children.last.path.last shows the path [2] and it's working with the path [2].